perl.c

```diff
@@ -2098,6 +2098,7 @@ S_parse_body(pTHX_ char **env, XSINIT_t xsinit)
         case 'c':
         case 'd':
         case 'D':
+        case 'g':
```

Comment:
This comment is a last ditch effort to dissuade the use of "-g", which is the VERY FIRST option listed in `gcc --help` under "debugging options" [1]; I strongly implore you all to use something that's more related to what this affects, namely "RS" (awk and is even explicitly referenced in `perlvar`).

My only "better": suggestion is lowercase "r", because it has to be a single character; and "r" may reflect "nothing" compared to the default "RS" of something (\n).

BUT, this about NOT using **-g** and going with something else. Literally anything else would be better than **-g** if there is even a sliver of concern regarding confusing and debugging options with those maybe more familiar with `gcc` than `perl`.

[1]
```
Debugging Options
           **-g**  -glevel  -gcoff  -gdwarf  -gdwarf-version -ggdb
           -grecord-gcc-switches  -gno-record-gcc-switches -gstabs  -gstabs+
           -gstrict-dwarf  -gno-strict-dwarf -gcolumn-info  -gno-column-info
           -gvms  -gxcoff  -gxcoff+  -gz[=type] -fdebug-prefix-map=old=new
           -fdebug-types-section -feliminate-dwarf2-dups
           -fno-eliminate-unused-debug-types -femit-struct-debug-baseonly
           -femit-struct-debug-reduced -femit-struct-debug-detailed[=spec-
           list] -feliminate-unused-debug-symbols  -femit-class-debug-always
           -fno-merge-debug-strings  -fno-dwarf2-cfi-asm -fvar-tracking
           -fvar-tracking-assignments
```

Reply:
As I said in the RFC discussion, collisions with single letter flags in other tools are unavoidable. For example, "-r" often means "recursive", "-o" means "output", etc. I don't think it's worth worrying about.

Reply:
I agree, the risk of confusion with a specific gcc flag seems minimal.

Reply:
Thank you for reconsidering it, which is what I was primarily asking you to do. I hope you both are correct and it ends up being a nothingburger on my part.